.github/workflows/add-action-project.yml

```diff
@@ -0,0 +1,16 @@
+name: Add new issues to project board
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  add-to-project:
+    if: github.repository_owner == 'cowprotocol'
+    name: Add issue to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.3.0
+        with:
+          project-url: https://github.com/orgs/cowprotocol/projects/8
+          github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
```

Comment:
How is this token generated?

Reply:
This is a personal access token (currently, it's simply a personal access token that I've generated for my account). Moving this to something not specific to my account would be preferable. It does need to be a semi-permissioned account, with admin rights in order to be able to write to the organization-wide project.

Reply:
After further discussion, we decided that keeping it running with the current setup because we couldn't find a better way to do this.